api/src/movie/dto/update-movie-list.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { Type } from 'class-transformer';
+import { IsArray, ValidateNested } from 'class-validator';
+import { MovieDto } from '../dto/movie.dto';
```

Comment:
Is this correct? aren't they supposed to be in the same directory? Should we put all the DTOs in the same file?

Reply:
Hm weird. Maybe this was a vscode issue that wasn't caught? not sure. But no I don't think we should put them all in the same file, seems messy.